great_expectations/expectations/core/expect_table_row_count_to_equal.py

```diff
@@ -1,6 +1,6 @@
 from __future__ import annotations
 
-from typing import TYPE_CHECKING, Any, ClassVar, Dict, List, Optional, Type, Union
```

Comment:
FYI, not that it matters: pretty sure we can just use `tuple` (lowercase, without importing) since we only support python 3.9+. Same with many of these types.